### DISCUSSION TITLE
Unable to use test-runner option in stryker4s config

### DISCUSSION THREAD

**Discussion by StasSuslov** [stryker-mutator/stryker4s#1556] [Q&A]
Hello!

I'm trying to use option `test-runner` in the `stryker4s.conf` file and while I'm launching stryker, I got this log message:

`[info] Attempting to read config from stryker4s.conf` 
`[warn] The following configuration key(s) are not used, they could stem from an older stryker4s version: 'test-runner'.`
`[warn] Please check the documentation at https://stryker-mutator.io/docs/stryker4s/configuration for available options.`
`[info] Attempting to read config from stryker4s.conf`

I tried legacy runner with `legacy-test-runner: true` but it didn't help. Can someone please give me a hint on how to work around this problem?
It's a bit funny though: the log says I should check the configuration page, where I found this runner config.

**Comment by hugo-vrijswijk** [comment]
Hi! What plugin of Strykers are you using? From your output I'm guessing sbt? The `test-runner` option is not a valid option for the sbt or maven plugin. It only works for the command-runner (which just executes the command). Are you getting any other exception or message that is preventing Stryker4s from running properly?

**Reply by StasSuslov** [reply]
Hi! Yep, that's right, I'm using `sbt-stryker4s` version `0.16.1`.
Oh, I see. Now my current flow is launch `sbt` and then run stryker with `stryker` cmd.
No, if I'm using plugin without any additional config everything's ok.
Btw, maybe you can give me a hit: I'd like to run Int tests instead of units. In case of this I tried `test-runner` option. Maybe there's a known way for this purpose?
